app/views/citationInfo.html

```diff
@@ -124,8 +124,8 @@ <h2 class="no-indent">Court Information</h2>
         </div>
         <div class="col-sm-6 alignColumnsToPage no-print">
           <h2 class="no-indent">I Want To...</h2>
-          <p>
-            <a ng-if="ctrl.paymentUrl != '' && ctrl.canPayOnline(ctrl.selectedCitation)" href="{{ctrl.paymentUrl}}" target="_blank">
+          <p ng-if="ctrl.paymentUrl != '' && ctrl.canPayOnline(ctrl.selectedCitation)">
```

Comment:
Please pull this logic into its own meaningful controller function that way you can write more unit tests around it since we don't have any UI functional tests.